scripts/set-versions.sh

```diff
@@ -11,17 +11,15 @@ readonly IBC_GO_DIR="ibc-go-src"
 readonly NEUTRON_DIR="neutron-src"
 readonly SLINKY_DIR="slinky-src"
 readonly WASMD_DIR="wasmd-src"
-readonly ICS_DIR="interchain-security-src"
 
-readonly ADMIN_MODULE_REV="v2.0.2"
+readonly ADMIN_MODULE_REV="feat/eth-sign"
 readonly BLOCK_SDK_REV="v2.1.5"
 readonly COSMOS_SDK_REV="v0.50.9-neutron"
 readonly FEEMARKET_REV="v1.1.1"
 readonly IBC_GO_REV="v8.5.1"
-readonly NEUTRON_REV="test/move-to-sovereign"
-readonly SLINKY_REV="v1.0.13"
+readonly NEUTRON_REV="feat/metamask"
```

Comment:
there's no PR for this branch in the neutron core repo. why? is everything up to date here?